pages/national_resources/gr_resources.md

```diff
@@ -1,65 +1,90 @@
-      your_tasks: [<!---REPLACE THIS with the page ID of the your_tasks pages that you want to list here as related pages--->]
-    url:
-    registry:
-      biotools: <!--- DELETE ME if not needed --->
-      fairsharing: <!--- DELETE ME if not needed --->
-      tess: <!--- DELETE ME if not needed --->
----
-<!---All the resources added above will appear on the table at the bottom of the page--->
 
-<!---Following information for the page text--->
-<!---Use this template as guidance, all fields are optional. Feel free to modify any section if you think it is necessary--->
-<!---If the information is already in another resource, please include the link instead of duplicating information--->
-<!---Please focus on resources that are relevant for the whole country for life sciences--->
+  - name: HYPATIA 
+    description:  HYPATIA is the Cloud infrastructure that has been developed to support the computational needs of the ELIXIR-GR community, but also the broader community of life scientists in Greece and abroad.
+    how_to_access:  Login via Life Science (LS) Login required.
+    url: https://hypatia.athenarc.gr/
+    
+  - name: Genomic Data Infrastructure (GDI)
+    description:  Infrastructure for secure genomic data management, including storage, discovery, access, and reception.
+    how_to_access:  Login via Life Science (LS) Login required to access ELIXIR-Greece GDI Portal.
+    url: https://login.gdi.elixir-greece.org/
+
+  - name: Greek COVID-19 Portal
+    description: Provides information, guidelines, tools and services to support researchers to utilise Greek and European infrastructures for data sharing. The portal is a national node of the European COVID-19 Data Portal.
+    url: https://covid19dataportal.gr/
+    related_pages: https://rdmkit.elixir-europe.org/covid19_data_portal
+
+ref_to_main_resources:
+  - argos
+---
 
 ## Introduction 
-<!---General RDM considerations for your country, how to deal with RDM on a national level--->
+This page provides useful information and resources with a focus on research data management in Greece. An overview of services provided by ELIXIR Greece can be found on the website https://www.elixir-greece.org/node/268.
 
-## Funders
+<!---## Funders--->
 
-## Regulations
+<!---## Regulations--->
 <!--- Ethical and legal regulations in the country, committees, etc. --->
 
+## Funders in Greece
+Research funding in Greece is primarily provided by **National and European funding agencies**. The major national funders supporting Open Science and Research Data Management (RDM) include:
+
+- The **[Hellenic Foundation for Research and Innovation (HFRI)](https://www.elidek.gr/)** which is a key funder supporting fundamental and applied research in Greece. 
+
+  HFRI Policies and Requirements:
+  - Encourages Open Science and FAIR data principles.
+  - Requires researchers to submit a Data Management Plan (DMP).
```

Comment:
at what stage it requires DMP submission, before/after/during project?

Reply:
@eleni-adamidi I think you should keep the requirement and mention that the time point at which the DMP is required may vary.